pkgs/applications/editors/jetbrains/plugins/specialPlugins.nix

```diff
@@ -0,0 +1,34 @@
+{ delve, autoPatchelfHook, stdenv }:
+# This is a list of plugins that need special treatment. For example, the go plugin (id is 9568) comes with delve, a
+# debugger, but that needs various linking fixes. The changes here replace it with the system one.
+{
+  "631" = {
+    # Python
+    nativeBuildInputs = [ autoPatchelfHook ];
+    buildInputs = [ stdenv.cc.cc.lib ];
+  };
+  "7322" = {
+    # Python community edition
+    nativeBuildInputs = [ autoPatchelfHook ];
+    buildInputs = [ stdenv.cc.cc.lib ];
+  };
+  "8182" = {
+    # Rust
+    nativeBuildInputs = [ autoPatchelfHook ];
+    commands = ''
```

Comment:
Hi @GenericNerdyUsername,

first of all: thanks for this great PR! 

While trying to get the Github Copilot plugin to work with your changes I noticed that the contents of the `commands` attribute are never evaluated as these values are folded onto the mkDerivation parameters in mkPlugin. It seems to be a leftover from your old, closed PR where you used this in attribute to construct a custom buildPhase. Am I missing something or should `buildPhase` be used here instead?

Reply:
yes, youre absolutely right, ill fix this straight away

Reply:
should be done, I built all the IDEs with the rust plugin but I might've missed something, lmk if the change breaks your copilot setup